src/ui/icon.ts

```diff
@@ -0,0 +1,7 @@
+// ICON.ts
+import { createVNode } from 'vue'
+import * as $Icon from '@ant-design/icons-vue'
+export const Icon = (props: { icon: string }) => {
+  // const { icon } = props
+  return createVNode($Icon[props.icon])
+}
```

Comment:
Woah, I didn't know one could create Vue.js components like that. I like it 

One tiny bit regarding ant-design and imports in general: As irritating as it is to manually list out every single import that one is using, there is a real reason to do so. It's useful for keeping the size of the final website down.
When we run `npm run build`, the build tool will check which things have been imported and only include those. 

Though, importing everything for now and only dealing with that later when we have a good idea of what we need is a totally fine strategy. And really, the extra MB or two from ant-design are peanuts compared to Pyodide and sympy.

Reply:
Honestly me neither, this is one of the few ways (hacks) to make a dynamic icon from Antdv 2.x.